app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/previous_return_exists_with_same_completion_date.scala.html

```diff
@@ -20,44 +20,69 @@
 
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
 @import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
-@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.UserType
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.TimeUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReturnSummary
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
 
 @this(
         mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
         pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
         backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
 )
 
-@(isATrust: Boolean, backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
+@(previousReturn: ReturnSummary, backLink: Call)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
 
 @key = @{"previousReturnExistsWithSameCompletionDate"}
-@userKey = @{
-    if(request.userType.contains(UserType.Agent)) ".agent"
-    else if(isATrust) ".trust"
-    else ""
+@title = @{ messages(s"$key.title") }
+@line1Display = @{
+    previousReturn.propertyAddress match {
+        case Address.UkAddress(line1, _, _, _, _) => line1
+        case Address.NonUkAddress(line1, _, _, _, _, _) => line1
+    }
 }
 
-
-@title = @{messages(s"$key.title")}
+@line2Display = @{
+    previousReturn.propertyAddress match {
+        case Address.UkAddress(_, _, _, Some(town), postcode) => s"$town, ${postcode.value}"
+        case _ => ""
```

Comment:
did you confirm how uk and non-uk addresses were supposed to be displayed? Are all lines supposed to be there or not?